.eslintrc

```diff
@@ -119,7 +119,7 @@ rules:
       enforceForArrowConditionals: false
 
   # Disallows unnecessary semicolons.
-  no-extra-semi: 2
+  no-extra-semi: 1
```

Comment:
FYI create react app changed the build process so that eslint errors blocked the build. To match the previous behavior, these were reduced to warnings.